README.md

```diff
@@ -113,6 +113,18 @@ wget https://www.ebi.ac.uk/metagenomics/api/v1/analyses/MGYA00575344/file/ERZ166
 wget https://www.ebi.ac.uk/metagenomics/api/v1/analyses/MGYA00575345/file/ERZ1664501_FASTA.fasta.gz
 ```
 
+## Broken Samplesheets
+
+We also hold 'broken' samplesheets for testing input schema validation.
+
+- `broken_samplesheets/samplesheet_missing_fasta.csv`: missing mandatory `fasta` column
+- `broken_samplesheets/samplesheet_missing_sample.csv`: missing entries in `sample` column
+- `broken_samplesheets/samplesheet_nonunique.csv`: duplicated row and duplicated single path
+- `broken_samplesheets/samplesheet_preannotated_invalidsample.csv`: missing sample entry and space in string
+- `broken_samplesheets/samplesheet_preannotated_missinggbk.csv`: missing `gbk` file when `protein` also given
+- `broken_samplesheets/samplesheet_preannotated_missingprotein.csv`: missing `protein` file when `gbk` also given
+- `broken_samplesheets/samplesheet_preannotated_missingsample.csv`: missing sample entry and space in string
```

Comment:
```suggestion
- `broken_samplesheets/samplesheet_preannotated_missingsample.csv`: missing sample entry
```
There is no space in sample string for this samplesheet. So either add space or adapt explanation like so.

Reply:
But not in this file (`samplesheet_preannotated_missingsample.csv`), only in `samplesheet_preannotated_invalidsample.csv` :see_no_evil: